smart_campus/app/views.py

```diff
@@ -2,24 +2,35 @@
 from django.views.decorators.csrf import csrf_exempt
 from django.views.decorators.http import require_POST
 from django.contrib import auth
-from django.http import HttpResponseRedirect, JsonResponse, HttpResponse, HttpResponseForbidden
+from django.http import (
+    HttpResponseRedirect,
+    JsonResponse,
+    HttpResponseForbidden,
+    HttpResponse
+)
 from django.shortcuts import get_object_or_404
 from django.contrib.auth.decorators import login_required
 from django.contrib import messages
 from django.contrib.gis.geos import Point
 from django.conf import settings
 from django.core.files.uploadedfile import InMemoryUploadedFile
 
+
 import os
 import random
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
Is there any reason that you initiate `logger` first instead of importing  3rd party libs first?

Reply:
None, my fault, removed it.